geomesa-plugin/src/main/resources/org/locationtech/geomesa/plugin/ui/js/entries-chart.js

```diff
@@ -17,10 +17,20 @@ function entriesChart() {
         .append("g")
         .attr("transform", "translate(" + margin.left + "," + margin.top + ")");
 
+    var suffixTickFormatter = function(n) {
+        var suffixes = ["", "K", "M", "G", "T", "P", "E"];
+        var idx = 0;
+        while (n >= 1000.0) {
+            idx += 1;
+            n = (1.0 * n) / 1000.0;
```

Comment:
[style] It feels like either this line or line 23 ought to change...  If ```n``` is ```Double```, then the multiplication by 1.0 shouldn't be necessary; if it's an ```Int```, then the test should be ```n >= 1000```, yes?